src/components/SignUpForm.js

```diff
@@ -0,0 +1,97 @@
+import { useEffect, useState } from "react";
+import InputField from "./InputField";
+import { useNavigate } from "react-router";
+import axios from "axios";
+
+function SignUpForm() {
+  const navigate = useNavigate();
+
+  const [name,setName] = useState("")
+  const [email,setEmail]= useState("")
+  const [password,setPassword] = useState("");
+  const [error,setError] = useState(true);
+
+
+ async function createUserAPI (){
+    try {
+      const body = {
+        name : name,
+        email : email,
+        password : password,
+      }
+      const headers = {
+        "Content-Type": "application/json"
+      }
+      const response = await axios.post("http://localhost:4000/api/users/", body, { headers: headers });
```

Comment:
```suggestion
      const response = await axios.post(`{BASE_URL}/api/users/`, body, { headers: headers });
```
We need to define this `BASE_URL` in `config/index.js` that will get its value from an env.